browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2719,9 +2710,12 @@ index d4e6b5d1f2151a1f4c16081b2695dd66da416ba2..7ddcea8b824043a6a17cfe2d209d861b
 +    m_frontendDispatcher->navigatedWithinDocument(frameId(&frame), url);
  }
  
--void InspectorPageAgent::didClearWindowObjectInWorld(Frame& frame)
+-void InspectorPageAgent::didClearWindowObjectInWorld(Frame& frame, DOMWrapperWorld& world)
```

Comment:
Why can't we use the same logic as upstream?

Reply:
We should not eval bootstrap scripts in two different locations. And they should not do that either.

Reply:
Agree with that, you can remove 
```
if (&world != &mainThreadNormalWorld())
    return;
```
check from this method and should work for us too with a smaller diff. Do you know btw why it's not evaluated in all worlds upstream?